dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/logic/DependsOnComponentValue.kt

```diff
@@ -12,7 +12,7 @@ import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkBoo
 class DependsOnComponentValue(var component: ComponentBase, var value: String) : FrameworkConditional() {
     override fun toFrameworkBooleanLambda(): FrameworkBooleanLambda {
         return FrameworkBooleanLambda(
-            "${component.getTypescriptFieldAccessor()} " +
+            "${component.getTypescriptFieldAccessor(true)} " +
```

Comment:
Doesn't this break stufff wherever the field is no datapoint?

Reply:
no, this was marcs suggestion.
it just recursively accesses the lowest field

Reply:
Da es aktuell keinen Schaden anrichtet und ich da auch kein Problem sehe. Let's go.